<launch>
    <arg name="headless" default="true"/>
    <arg name="gui" default="false"/>
    <arg name="enable_logging" default="false"/>
    <arg name="enable_ground_truth" default="false"/>
    <arg name="log_file" default="iris"/>

    <include file="$(find px4)/launch/gazebo_iris_empty_world.launch">
        <arg name="headless" value="$(arg headless)"/>
        <arg name="gui" value="$(arg gui)"/>
        <arg name="enable_logging" value="$(arg enable_logging)" />
        <arg name="enable_ground_truth" value="$(arg enable_ground_truth)" />
        <arg name="log_file" value="$(arg log_file)"/>
    </include>

    <test test-name="direct_manual_input_test" pkg="px4" type="direct_manual_input_test.py" />
    <test test-name="direct_offboard_posctl_test" pkg="px4" type="direct_offboard_posctl_test.py" />
</launch>
